client/src/views/header/HeaderPage.vue

```diff
@@ -265,8 +272,16 @@ async function openNotificationCenter(event: Event): Promise<void> {
   });
   await popover.present();
   await popover.onWillDismiss();
+  notificationPopoverIsVisible.value = false;
   await popover.dismiss();
 }
+
+function hasUnreadNotifications(): boolean {
```

Comment:
notificationManager already has a 'hasUnreadNotifications()' function you don't need to add it here.

Reply:
Done